xc/device.ml

```diff
@@ -1384,6 +1384,23 @@ module PCI = struct
     write_string_to_file new_slot devstr;
     write_string_to_file bind devstr
 
+  let unbind_from_pciback devstr =
+    let rm_slot = Filename.concat sysfs_pciback "remove_slot" in
+    let unbind = Filename.concat sysfs_pciback "unbind" in
+    let device = Filename.concat sysfs_devices devstr  in
+    let driver = Filename.concat device "driver" in
+    let has_pciback =
+      try
+        Unix.readlink driver |> Filename.basename |> (=) "pciback"
```

Comment:
There is a `get_driver` function above here, which does about the same as the new lines up to here. In fact, we have used `get_driver` in `bind`, which is how `unbind_from_pciback` got called from there, so the check is done twice on that code path.